### DISCUSSION TITLE
how to get data in  td:nth-child(5)

### DISCUSSION THREAD

**Discussion by Srikarthikeyan4006** [tusharojha/web_scraper#41] [Q&A] [answered]

**Comment by tusharojha** [comment, marked as answer]
You can add td:nth-child(5) inside the selector to do it.
For example:
`div > section > ul:nth-child(2) > li:nth-child(1)`

**Reply by Srikarthikeyan4006** [reply]
i am unable to do ..can you please help me to get only address for this site https://ngodarpan.gov.in/index.php/home/statewise_ngo/7589/33/1?per_page=100

**Reply by tusharojha** [reply]
Try this:

```dart
void loadPage() async {
  if (await webScraper.loadFullURL(
      'https://ngodarpan.gov.in/index.php/home/statewise_ngo/7589/33/1?per_page=100')) {
    try {
      var elements = webScraper.getElementTitle('tbody > tr > td:nth-child(4)');
      elements.forEach((img) => print(img));
    } catch (e) {
      print(e);
    }
  }

```

**Reply by Srikarthikeyan4006** [reply]
but when i gave td:nth-child(5) i am unable to get  "Sectors working in" any idea how to get it

**Comment by Srikarthikeyan4006** [comment]
thanks a lot ..it worked well
